docker-compose.yml

```diff
@@ -2,23 +2,22 @@ version: '2'
 services:
   app:
     build: .
+    platform: linux/arm64/v8
     volumes:
       - .:/usr/src/app
-      - node_modules:/usr/src/app/node_modules
+      # - node_modules:/usr/src/app/node_modules
     links:
       - consul
     environment:
       NODE_ENV: development
       CONSUL_HOST: "consul"
       CONSUL_PORT: "8500"
-      CONSUL_JWT_SECRET_PREFIX: "config/development/secrets"
-
+      CONSUL_JWT_SECRET_PREFIX: "/config/development/secrets"
```

Comment:
I would add a `depends_on` line to this config to ensure the consule service is started before the app.